react/src/lib/List/examples/KitchenSink.js

```diff
@@ -1,17 +1,34 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React from 'react';
 import {
   ListDefault,
   ListListItemHeader,
   ListListItemSeparator,
+  CheckBoxForKeyDownEvents,
 } from './index';
 
 export default class ListKitchenSink extends React.Component {
+  state ={
```

Comment:
Can we move this to the individual example instead of the KitchenSink.

Reply:
Moved to new component,